paper/joss_preview/paper.Rmd

```diff
@@ -0,0 +1,165 @@
+- uncertainty is more precise and reliable.  
+- uncertainty around survival estimates can be estimated in cases with 0 mortalities in a year.  
+
+In addition, with a Bayesian framework:  
+- models can incorporate prior knowledge, which is especially useful when data are sparse.  
+- posterior distributions of parameters can be combined to produce derived parameters (e.g., population growth) with full information about uncertainty retained.  
+
+A `bboutools` survival model including a year random effect and trend takes the form
+$$
+\begin{aligned}
+d_{ij} & \sim \text{Binomial}(1 - \widehat{S_{ij}}, r_{ij}) \\
+\text{logit}(\widehat{{S}_{ij}}) & = \beta_{0} + \alpha_{i} + \delta_{j} + \beta_{1} \cdot Year_{j} \\
+\alpha_{i} & \sim \text{Normal}(0, \sigma_{\alpha}) \\
+\delta_{j} & \sim \text{Normal}(0, \sigma_{\delta}) \\
+\end{aligned}
+$$
+
+where $d_{i,j}$ is the number of mortalities in the $i^{th}$ month and $j^{th}$ year, $r_{i,j}$ is the number of collared individuals at the start of the month, $\widehat{{S}_{ij}}$ is the survival probability, $\alpha_i$ is a monthly random effect with SD $\sigma_{\alpha}$, $\delta_i$ is an annual random effect with SD $\sigma_{\delta}$ and $\beta_{1}$ represents the trend. 
+
+A `bboutools` recruitment model including annual random effect and trend takes the form
+$$
+\begin{aligned}
+Calves_{j} & \sim \text{Binomial}(X_{j}, AdultFemales_{j}) \\
+\text{logit}(\delta_{j}) & = \beta_{0} + \alpha_{j} + \beta_{j} \cdot Year_{j} \\
+\alpha_{j} & \sim \text{Normal}(0, \sigma_{\alpha}) \\
+FemaleYearlings_{j} & \sim \text{Binomial}(sex\_ratio, Yearlings_{j}) \\
+Cows_{j} & \sim \text{Binomial}(adult\_sex\_ratio, CowsBulls_{j}) \\
+AdultFemalesOther_{j} & \sim \text{Binomial}(adult\_sex\_ratio, AdultsUnknown_{j}) \\
+AdultFemales_{j} & = FemaleYearlings_{j} + Cows_{j} + AdultFemalesOther_{j} \\
+\end{aligned}
+$$
+
+where $X_{j}$ is the calves per adult female in the $j^{th}$ year, $\alpha_{j}$ is an annual random effect with SD $\sigma_{\alpha}$ and $\beta_{1}$ represents the trend. 
+
+The model includes demographic stochasticity. 
+Groups are aggregated by year prior to model fitting.
+The sex ratio is fixed and can be adjusted by the user, with default of 0.5.
+The adult female proportion can be estimated from counts of cows and bulls or fixed, with a default of 0.65, which accounts for higher mortality of males [@smith_2004].
+
+We compared estimates from `bbouretro` and `bboutools` on anonymized empirical and simulated data.
+
+[Empirical comparisons](https://poissonconsulting.github.io/bbousuite/articles/empirical-comparisons.html) demonstrate that `bbouretro` methods yield similar estimates to Bayesian fixed-effects models with uninformative priors (\autoref{fig:empirical}).
+By default, `bboutools` uses uninformative priors and these can be adjusted by the user. 
+Estimates from random effects models tend to be less extreme than traditional and fixed effect models.
```

Comment:
```suggestion
Estimates from random effects models tend to be provide more reliable estimates than traditional and fixed effect models.
```

Reply:
I'm unsure about this change. This paragraph is referring to the empirical comparison (not simulated data). My intention was to point out that random effect estimates are closer to the mean than the fixed effect estimates. However at this point I'm not sure the empirical comparison can show us that the random effect estimates are more reliable - that comes further down with simulations

Reply:
although I agree "tend to be less extreme" doesn't sound great either...

Reply:
We could go with "less noisy"?

Reply:
yes sounds good